### PR TITLE
add icds-new env

### PR DESCRIPTION
**Pull request by sravfeyn**
The agg tests fail locally for me without this.

### CODE REVIEW

**Comment by calellowitz** [comment]
Can we change the test settings instead so they use `icds` as the env

**Comment by sravfeyn** [comment]
We could, I was just trying to follow the rest of the UCR sources.

**Comment by sravfeyn** [comment]
@calellowitz updated.

File: custom/icds_reports/ucr/data_sources/person_cases_v3.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
   ],
   "server_environment": [
     "softlayer",
-    "icds"
+    "icds",
+    "icds-new"
```
Comment (emord) [inline custom/icds_reports/ucr/data_sources/person_cases_v3.json]: You should be able to remove this now.

Comment (sravfeyn) [reply custom/icds_reports/ucr/data_sources/person_cases_v3.json]: FWIW, it's consistent with most of other PRs.

Comment (emord) [reply custom/icds_reports/ucr/data_sources/person_cases_v3.json]: Then can you remove it from the other UCRs? we no longer  run an icds-new environment so I think it makes sense to not add new code that references it

File: custom/icds_reports/ucr/data_sources/person_cases_v3.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
   ],
   "server_environment": [
     "softlayer",
-    "icds"
+    "icds",
```
Comment (emord) [inline custom/icds_reports/ucr/data_sources/person_cases_v3.json]:
```
ERROR: corehq.apps.userreports.tests.test_static_data_sources:TestStaticDataSource.test_for_table_id_conflicts
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vendor/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/tests/test_static_data_sources.py", line 64, in test_for_table_id_conflicts
    StaticDataSourceConfiguration.all())
  File "/usr/local/lib/python2.7/collections.py", line 477, in __init__
    self.update(*args, **kwds)
  File "/usr/local/lib/python2.7/collections.py", line 566, in update
    for elem in iterable:
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/tests/test_static_data_sources.py", line 63, in <genexpr>
    counts = Counter((ds.table_id, ds.domain) for ds in
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/models.py", line 700, in all
    for wrapped in cls._all():
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/models.py", line 684, in __get_all
    yield _get_wrapped_object_from_file(path_or_glob, cls)
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/models.py", line 1099, in _get_wrapped_object_from_file
    doc = json.load(f)
  File "/usr/local/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/local/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
----------------------------------------------------------------------
```

hanging commas aren't syntactically correct for JSON. you can run that above test to find these kinds of issues

Comment (sravfeyn) [reply custom/icds_reports/ucr/data_sources/person_cases_v3.json]: Thanks, it was a typo.

**Comment by sravfeyn** [comment]
@emord

**Pull request merged**